lib/kvdb/ikvdb.c

```diff
@@ -2151,9 +2151,9 @@ ikvdb_kvs_count(struct ikvdb *handle, unsigned int *count)
 }
 
 merr_t
-ikvdb_kvs_query_tree(struct hse_kvs *kvs, struct yaml_context *yc, bool list)
+ikvdb_kvs_query_tree(struct hse_kvs *kvs, struct yaml_context *yc, bool blkids, bool nodesonly)
```

Comment:
maybe change `bool nodesonly` to `bool kvsets` so `false, false` is minimal output and `true, true` is maximal output?

alternatively, since showing blkids depends on showsing kvsets, it would be logical to use `int verbosity` where 0 means nodes only, 1 means nodes+kvsets, 2 means nodes+kvsets+blkids

Reply:
Another PR?  The only reason this exists is to support ikvdb_kvs_query_tree() which is only called by cn_metrics.c in order to output the tree in yaml.  Does anybody use that interface???

Reply:
In my PR, I am planning to remove this hack from cn_metrics.c. Hate how it calls into the REST code :)